### DISCUSSION TITLE
https://www.dcm4che.org/maven2 seem to be down

### DISCUSSION THREAD

**Discussion by Homersbm** [dcm4che/dcm4che#983] [Q&A]
I am using  	dcm4che-json  version 5.23.1,  dcm4che-core   and dcm4che-mime version 5.22.0. Looks like the repository https://www.dcm4che.org/maven2 is down. Has this repository been moved to Maven ??

**Comment by hczedik** [comment]
The whole dcm4che.org domain is not working at the moment unfortunately.

For a workaround please see https://github.com/dcm4che/dcm4che/issues/975

**Reply by Homersbm** [reply]
I am working on windows. Is there any work around you could suggest ?
I kept "195.201.27.249 www.dcm4che.org" in c:\Windows\System32\Drivers\etc\hosts file .. as there is a suggestion in the comments.
but i see error
"DNS error (the host name of the page you are looking for does not exist)". I would like to download certain libraries and work locally until the site is back.

**Comment by Homersbm** [comment]
please refer to this Repository : https://maven.scijava.org/content/repositories/public/
